numpy/testing/tests/test_utils.py

```diff
@@ -1114,18 +1114,26 @@ def test_simple(self):
                           lambda: assert_string_equal("foo", "hello"))
 
 
-def assert_warn_len_equal(mod, n_in_context, py3_n_in_context=None):
+def assert_warn_len_equal(mod, n_in_context, py34=None, py37=None):
     mod_warns = mod.__warningregistry__
+    num_warns = len(mod_warns)
     # Python 3.4 appears to clear any pre-existing warnings of the same type,
```

Comment:
Is this comment still in the right place?

Reply:
Beats me, it's not my comment and I don't really know what it means :-)

Reply:
I don't know what it means either, but hoped you might :)